libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -955,6 +955,70 @@ def SetTags(self, new_tags: List[str]) -> None:
     response = request.execute()
     self.BlockOperation(response, zone=self.zone)
 
+  def GetPowerState(self) -> str:
+    """
+    Gets the current power state of the instance.
+
+    As per https://cloud.google.com/compute/docs/reference/rest/v1/instances/get
+    this can return one of the following possible values: PROVISIONING, STAGING,
+    RUNNING, STOPPING, SUSPENDING, SUSPENDED, REPAIRING, and TERMINATED
+    """
+    return str(self.GetOperation()['status'])
+
+  def Stop(self) -> None:
+    """
+    Stop the instance.
```

Comment:
We try to stick to descriptive rather than imperative docstrings ([details here](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods))

```suggestion
    Stops the instance.
```